.github/workflows/ci.yaml

```diff
@@ -476,7 +495,6 @@ jobs:
           tar xvzf updater-build.tgz
 
       - uses: actions/download-artifact@v4
-        if: ${{ ! startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Did this condition cause the subsequent steps to fail if it was run for a tag?

Reply:
I don't recall specifically why I removed this, but it looks like this job (`diff-dumps`) won't/shouldn't run on tags anyway.